index.html

```diff
@@ -2,7 +2,21 @@
 <html>
   <head>
     <title>Memory exercise</title>
+    <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+    <script src="game.js"></script>
+    <script src="board.js"></script>
+    <script src="tile.js"></script>
+    <script src="memory.js"></script>
+    <link rel="stylesheet" type="text/css" href="memory.css">
   </head>
   <body>
+    <div id="board-config">
+        <input id="board-size" placeholder="Enter Size" />
```

Comment:
See comment above: you could probably skip asking for this, and just use the other two numbers to calculate it.